doc/api/path.markdown

```diff
@@ -105,28 +105,41 @@ and the `ext` property will be used as the `base` property.
 
 Examples:
 
-An example on Posix systems:
+Some Posix system examples:
 
 ```js
+// if `dir` and `base` are provided, `dir` + platform separator + `base`
+// will be returned
 path.format({
-    root : "/",
-    dir : "/home/user/dir",
-    base : "file.txt",
-    ext : ".txt",
-    name : "file"
+    dir: '/home/user/dir',
+    base: 'file.txt'
 });
 // returns '/home/user/dir/file.txt'
 
-// `root` will be used if `dir` is not specified and `name` + `ext` will be used
-// if `base` is not specified
+// `root` will be used if `dir` is not specified
+// `name` + `ext` will be used if `base` is not specified
```

Comment:
Nit: Would it help to put a period at the end of this line and the one above to make it clear that the subsequent line is a new sentence?